hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -4039,7 +4031,7 @@ private static Get toGet(final Mutation mutation) throws IOException {
       // Sort the cells so that they match the order that they appear in the Get results.
       // Otherwise, we won't be able to find the existing values if the cells are not specified
       // in order by the client since cells are in an array list.
-      sort(deltas, store.getComparator());
+      deltas.sort(store.getComparator());
```

Comment:
not related?

Reply:
HRegion.sort is also referenced in RegionScannerImpl and it is just a one line function, so just removed it.